### PR TITLE
Scc 588

### PR DESCRIPTION
**Pull request by LaxmanDP**

### CODE REVIEW

File: test/controllers/IndexControllerSpec.scala [hunk 1]
```diff
@@ -139,82 +158,67 @@ class IndexControllerSpec extends SpecBase with BeforeAndAfterEach {
...
-          )
+             }
 
-          val application = applicationBuilder(userAnswers = Some(userAnswers))
-            .overrides(
-              bind[ChocsService].toInstance(mockService),
-              bind[SessionRepository].toInstance(mockRepository)
-            )
-            .build()
+              "must display 'Unknown' email address" in {
```
Comment (owen-opencast) [inline test/controllers/IndexControllerSpec.scala:190]: Can we add a test for Unknown in the telephone field in the style of the test for e-mail?

File: app/viewmodels/checkAnswers/TelephoneSummary.scala [hunk 2]
```diff
@@ -20,23 +20,22 @@ import controllers.routes
...
+  def row(answers: UserAnswers)(implicit messages: Messages): Seq[SummaryListRow] ={
+    val telephone = answers.get(TelephonePage).getOrElse(messages("index.personaldetails.unknown"))
+
+    Seq( SummaryListRowViewModel(
           key     = "index.personaldetails.label.telephone",
-          value   = ValueViewModel(HtmlFormat.escape(answer).toString),
+          value   = ValueViewModel(telephone),
           actions = Seq(
             ActionItemViewModel("site.change", routes.IndexController.onPageLoad.url)
               .withVisuallyHiddenText(messages("preferredName.change.hidden"))
```
Comment (owen-opencast) [inline app/viewmodels/checkAnswers/TelephoneSummary.scala:37]: I've just spotted the Visually Hidden Text is the same for all fields: preferredName.change.hidden
While we're in here, can we fix these, and update the messages file as needed (*.change.hidden)

**Review by owen-opencast — CHANGES_REQUESTED** [review]

File: test/controllers/IndexControllerSpec.scala [hunk 3]
```diff
@@ -139,82 +158,79 @@ class IndexControllerSpec extends SpecBase with BeforeAndAfterEach {
...
-            val result = route(application, request).value
+              }
 
-            status(result) mustEqual OK
+              "must display 'Unknown' phone number" in {
 
-            val pageContent = contentAsString(result)
-            val document: Document = Jsoup.parse(pageContent)
+                val contactDetails: Elements =
+                  document.select("#contact-details-list .email-address .govuk-summary-list__value")
```
Comment (owen-opencast) [inline test/controllers/IndexControllerSpec.scala:206]: I think .email-address should be .telephone here

**Review by owen-opencast — CHANGES_REQUESTED** [review]

File: test/controllers/IndexControllerSpec.scala (hunk 3 above)
Comment (LaxmanDP) [reply test/controllers/IndexControllerSpec.scala]: done

**Review by owen-opencast — APPROVED** [review]

**Pull request merged**